cloudinstall/single_install.py

```diff
@@ -178,7 +223,8 @@ def _install_upstream_deb(self):
         log.debug('Found upstream deb, installing that instead')
         filename = os.path.basename(self.config.getopt('upstream_deb'))
         utils.container_run(
-            self.container_name, 'sudo dpkg -i .cloud-install/{}'.format(
+            self.container_name,
+            'sudo dpkg -i /home/ubuntu/.cloud-install/{}'.format(
```

Comment:
yea just from watching the screen it looked about the same response back so I'll just leave it, seems a little more pure anyway.